server/service/testdata/apns.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think these files are leftovers from the merge. They got removed. Sorry about that!

Reply:
Ah no worries!